Formula/tesseract-lang.rb

```diff
@@ -0,0 +1,25 @@
+class TesseractLang < Formula
+  desc "Enables extra languages support for Tesseract"
+  homepage "https://github.com/tesseract-ocr/tessdata_fast/"
+  url "https://github.com/tesseract-ocr/tessdata_fast/archive/4.0.0.tar.gz"
+  sha256 "f1b71e97f27bafffb6a730ee66fd9dc021afc38f318fdc80a464a84a519227fe"
+
+  depends_on "tesseract"
+
+  def install
+    rm "eng.traineddata"
+    rm "osd.traineddata"
+    (share/"tessdata").install Dir.glob("./*")
+  end
+
+  test do
+    resource "tests" do
```

Comment:
- The resource should be defined at the top-level, not inside `test` block.
- Let's not include the whole repo as resource, but only the file you really need (`eurotext.tif`).